x/market/client/cli/lease.go

```diff
@@ -59,7 +59,9 @@ func cmdGetLease() *cobra.Command {
 		Short: "Query order",
 		Args:  cobra.ExactArgs(0),
 		RunE: func(cmd *cobra.Command, args []string) error {
+
 			clientCtx := client.GetClientContextFromCmd(cmd)
+			fmt.Printf("clientCtx: %+v\n", clientCtx)
```

Comment:
```suggestion
```

Reply:
Good catch, I forgot I was debugging this code. I'll remove it.